### PR TITLE
:sparkles: Add atst SDK

### PR DESCRIPTION
**Pull request by roninjin10**
Add the sdk to @eth-optimism/sdk

### CODE REVIEW

**Comment by roninjin10** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #4817** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4817" target="_blank">Graphite</a> 
    * **PR #4816** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4816" target="_blank">Graphite</a> 
      * **PR #4815** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4815" target="_blank">Graphite</a> 
        * **PR #4958** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4958" target="_blank">Graphite</a> 
          * **PR #4959** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4959" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/optimism/4959?utm_source=stack-comment).

File: packages/atst/src/constants/defaultDataType.ts
Comment (nickbalestra) [inline packages/atst/src/constants/defaultDataType.ts:5]: shouldn't for this make more sense to be declared within `types/DataTypeOption` as it seams is the default of that?

**Review by nickbalestra — APPROVED** [review]
LGTM!! Love the tests, best documentation ever imho! I see some tests failing in CI, is that expected or related ? Approving to unblock, just make sure to address those

File: packages/atst/src/constants/defaultDataType.ts
Comment (roninjin10) [reply packages/atst/src/constants/defaultDataType.ts:5]: Yea I think you are right I will leave this comment open and do this before landing

**Pull request merged**
